client/app/calendar/tasks/create/page.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
+
+import React from 'react'
+import { FormContainer, Loading } from '@/_components/layout'
+import { Button, Form, Heading } from '@/_components/ui'
+import { useGetHouseQuery } from '@/_hooks/api'
+import { AssigneeField, DateField, NoteField, TitleField } from '../components'
+import { useCreateTask } from '../hooks'
+import { TaskFormType } from '../types'
+
+export default function CreateTask() {
+  const { form, onSubmit: onCreateTask, isPending } = useCreateTask()
+
+  const { data: house, isLoading } = useGetHouseQuery()
+  const houseMembers = house?.houseMembers || []
+
+  if (isLoading) return <Loading />
+
+  return (
+    <FormContainer>
+      <Heading title='Create Task' />
+      <Form {...form}>
+        <form onSubmit={onCreateTask} className='mx-auto mt-8 w-full'>
+          <div className='space-y-5'>
+            <TitleField<TaskFormType> control={form.control} name='title' />
+            <DateField<TaskFormType> control={form.control} name='date' />
+            <AssigneeField<TaskFormType>
+              control={form.control}
+              name='assigneeId'
+              houseMembers={houseMembers}
+            />
+            <NoteField<TaskFormType> control={form.control} name='note' />
+          </div>
+          <div className='mt-10 text-center'>
+            <Button type='submit' isLoading={isPending} disabled={isPending}>
```

Comment:
I think you don't need to pass disabled because isLoading is showing meanwhile!

Reply:
Isn’t it better to still have disabled since there is a chance users can still click more than once even if they are seeing loading?

Reply:
Hmm you can't click while `isLoading` because I also define `pointer-events-none`. I still don't think passing disabled has meaning 🤔 Sorry if I get your answer wrongly

Reply:
Oh yeah, sorry I wasn't aware that there is pointer-events-none in Button component. In this case, I agree it's enough to prevent users to click while loading.

Reply:
Sorry, I actually just tried clicking many times but pointer-events-none seems not enought to prevent clicking many times, but with disabled it blocks clicking perfectly so I think we need disabled